fe/fe-core/src/main/java/org/apache/doris/analysis/StmtType.java

```diff
@@ -0,0 +1,61 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.analysis;
+
+public enum StmtType {
+    DDL,
```

Comment:
add comment
1. clarifying the criteria for classifying a statement. for example we have DDL and CREATE, so when to use DDL directly, when to add a new type of DDL?
2. sort and classify these types for easy read and maintain

Reply:
1.The reason for not simplifying the DDL type is that we can see more detailed SQL types. The reason for adding the DDL type is that if there are new DDL operations in the future, if there is no corresponding method implemented, it will be directly a DDL type.
2.done